std/string.d

```diff
@@ -2986,20 +2986,19 @@ if (isForwardRange!Range && isSomeChar!(ElementEncodingType!Range) &&
             // Fall through to standard case.
         }
 
-        static if (ElementType!Range.sizeof > ElementEncodingType!Range.sizeof)
+        static if (isNarrowString!Range)
         {
-            // Type performs its own decoding.
-            while (!input.empty)
+            import std.utf : decode, UseReplacementDchar;
```

Comment:
Duplicate of line 3002. Maybe just move the imports out of the `static if { .. } else { .. }` blocks, since it's in both.

Reply:
Done.